src/bin/dist.ts

```diff
@@ -1,23 +1,49 @@
 #!/usr/bin/env node
 
+import * as _ from "lodash";
 import config from "../config/config";
 import {ScriptLinter} from "../scripts";
 import {StyleCompiler, StyleLinter} from "../styles";
 import {TranslationCompiler} from "../translations";
 import {Assets} from "../assets";
+import {StylesCompileConfig} from "../config/config.interface";
 
-const onError = () =>
-  process.exit(1);
+const onError = () => process.exit(1);
 
-const assets = new Assets(config.assets.dist).on("error", onError);
+let assets: Assets;
 
-assets
-  .clean()
-  .on("clean", () => {
+const startTooling = () => {
```

Comment:
There is a lot of repetitive code inside the two `startTooling` functions - the only relevant difference I can see here is about accessing `.dist` instead of `.dev` in production mode. Although this is reasonable (since they fit the same interface), I'd suggest its better to use a common function to handle the particular config parts. This will reduce the required maintenance effort.